utils/IrodsConnector.py

```diff
@@ -137,7 +139,12 @@ def default_resc(self):
             Resource name.
 
         """
-        return self.ienv.get('irods_default_resource', None)
+        #irods 4.3.X
+        if 'irods_default_resource' in self.ienv:
+            return self.ienv.get('irods_default_resource', None)
+        #irods 4.2.X
+        if 'default_resource_name' in self.ienv:
+            return self.ienv.get('default_resource_name', None)
```

Comment:
`irods_default_resource` is the variable holding the default resource name in a user's `~/.irods/irods_environment.json`.  `default_resource_name` is the variable holding the same value on the server in the `server_config.json` file.  These are valid for all versions of iRODS as far as I'm aware.  Thus, this change is not needed.

For reference, please see:

https://docs.irods.org/4.2.11/system_overview/configuration/

and:

https://docs.irods.org/4.3.0/system_overview/configuration/
https://docs.irods.org/4.3.0/getting_started/installation/